src/components/profile/joinedMissions.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+const Joined = () => {
+  const missions = useSelector((state) => state.missions);
+  const joined = missions.filter((mission) => mission.reserved);
+  const reserveMissionContainer = {
+    margin: '30px auto',
+    display: 'flex',
+    flexDirection: 'column',
+  };
+  const table = {
+    width: '48%',
+    height: '50px',
+    borderCollapse: 'collapse',
+    border: '1px solid #AAAAAA',
+  };
+  return (
+    <div className="profile_container" style={reserveMissionContainer}>
+      <span style={{ margin: '20px 0', padding: '0' }}>
+        <h1>My Missions</h1>
+      </span>
+      <ul>
+        {joined.length > 0 ? (
+          joined.map((mission) => (
+            <table border={1} style={table} key={mission.mission_id}>
+              <tbody>
```

Comment:
Missions and rockets are literally the same UI, we only change the data.
What do you think if we created a separate table component that receives data as props.
ie... if we have `<TableData/>`, we can use it twice like that : `<TableData missions={missions}/>` `<TableData rockets={rockets}/>`

Reply:
Yeah it's okay bro
No problem
Let's do it